x/staking/keeper/validator.go

```diff
@@ -445,3 +445,12 @@ func (k Keeper) UnbondAllMatureValidators(ctx sdk.Context) {
 		}
 	}
 }
+
+func (k Keeper) IsJailed(ctx sdk.Context, addr sdk.ConsAddress) bool {
```

Comment:
The name of the function is overloaded. There is already a function with the same name in staking/types/validator.go. Do we actually need this function?

Reply:
Yes we do need this function it satisfies the `StakingKeeper` interface in x/slashing/types/expected_keepers.go. However we can change its name.